vector/src/main/java/im/vector/app/features/settings/notifications/VectorSettingsNotificationPreferenceFragment.kt

```diff
@@ -134,7 +134,8 @@ class VectorSettingsNotificationPreferenceFragment @Inject constructor(
             it.onPreferenceChangeListener = Preference.OnPreferenceChangeListener { _, newValue ->
                 if (newValue is String) {
                     val syncDelay = tryOrNull { Integer.parseInt(newValue) } ?: BackgroundSyncMode.DEFAULT_SYNC_DELAY_SECONDS
-                    vectorPreferences.setBackgroundSyncDelay(maxOf(0, syncDelay))
+                    // Tchap: Force a minimum value: 10 sec
+                    vectorPreferences.setBackgroundSyncDelay(maxOf(syncDelay, 10))
```

Comment:
can you please flip parameters to have minValue first?
```suggestion
                    vectorPreferences.setBackgroundSyncDelay(maxOf(10, syncDelay))
```

Reply:
You can also define it as a const to have default values at the same place